client/src/test/java/com/defold/extender/client/ExtenderClientTest.java

```diff
@@ -358,7 +364,12 @@ public CloseableHttpResponse execute(HttpUriRequest request) {
             final String HDR_VALUE_2 = "my custom header2";
             MockHttpClient httpClient = new MockHttpClient();
             File cacheDir = new File("build");
-            ExtenderClient extenderClient = new ExtenderClient(httpClient, "http://localhost", cacheDir);
+            Class<?> mockExtenderClientClass = Class.forName("com.defold.extender.client.ExtenderClient");
```

Comment:
Because of remove constructor with `httpClient` as argument we need to workaround and set private field in mocked instance.